sfs/util.py

```diff
@@ -555,6 +556,28 @@ def spherical_hn2(n, z):
     return spherical_jn(n, z) - 1j * spherical_yn(n, z)
 
 
+def jinc(x):
+    r"""Circular counterpart of Sinc function a.k.a. Jinc function
```

Comment:
The third dot messes up the short description used in the summary table.

Also:

```
$ python3 -m pydocstyle sfs/util.py
sfs/util.py:559 in public function `jinc`:
        D400: First line should end with a period (not 'n')
```